quotacontrol.go

```diff
@@ -228,10 +293,41 @@ func (q qcHandler) DisableAccessKey(ctx context.Context, accessKey string) (bool
 	if err != nil {
 		return false, err
 	}
+
+	list, err := q.accessKeyStore.ListAccessKeys(ctx, access.ProjectID, proto.Ptr(true), nil)
+	if err != nil {
+		return false, err
+	}
+
+	if len(list) == 1 {
+		return false, fmt.Errorf("There should be atleast one active accessKey per project")
+	}
+
+	isDefault := access.Default
+
 	access.Active = false
+	access.Default = false
 	if _, err := q.accessKeyStore.UpdateAccessKey(ctx, access); err != nil {
 		return false, err
 	}
+
+	if isDefault == false {
+		return true, nil
+	}
+
+	// set another project accessKey to default
+	listUpdated, err := q.accessKeyStore.ListAccessKeys(ctx, access.ProjectID, proto.Ptr(true), nil)
+	if err != nil {
+		return false, err
+	}
+
+	anotherAccessKey := listUpdated[0] 
+	if anotherAccessKey != nil {
```

Comment:
This should always be not nil, just use `listUpdated[0]` in the `UpdateDefaultAccessKey`